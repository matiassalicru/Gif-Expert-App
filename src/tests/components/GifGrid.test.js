import React from 'react';
const { shallow } = require("enzyme")
const { GifGrid } = require("../../components/GifGrid")


describe('Pruebas de GifGrid', () => {
    test('DeberÃ­a matchear con el snapshot ', () => {

        const category = 'algo';

        const wrapper = shallow(<GifGrid category={category}/>);

        expect(wrapper).toMatchSnapshot();
        
    })
    
    
})
