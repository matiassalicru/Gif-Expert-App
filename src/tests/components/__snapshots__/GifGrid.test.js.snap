// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas de GifGrid Deber√≠a matchear con el snapshot  1`] = `
<Fragment>
  <h3
    className="category"
  >
    algo
  </h3>
  <p
    className="animate__animated animate__flip"
  >
    Loading
  </p>
  <div
    className="card-grid"
  />
</Fragment>
`;
